blueflood-core/src/main/java/com/rackspacecloud/blueflood/rollup/Granularity.java

```diff
@@ -171,29 +171,55 @@ public Range deriveRange(int slot, long referenceMillis) {
     }
 
     /**
+     * Return granularity that maps most closely to requested number of points,
+     * using the algorithm specified in the
+     * {@code GET_BY_POINTS_SELECTION_ALGORITHM} config value. See
+     * {@link #granularityFromPointsInInterval(String, long, long, int, String, long)}.
+     *
+     * @param from beginning of interval (millis)
+     * @param to end of interval (millis)
+     * @param points count of desired data points
+     * @return
+     */
+    public static Granularity granularityFromPointsInInterval(String tenantid, long from, long to, int points) {
+        return granularityFromPointsInInterval(tenantid, from, to, points, GET_BY_POINTS_SELECTION_ALGORITHM, GET_BY_POINTS_ASSUME_INTERVAL, true);
+    }
+    /**
      * Return granularity that maps most closely to requested number of points based on
      * provided selection algorithm
      *
      * @param from beginning of interval (millis)
      * @param to end of interval (millis)
      * @param points count of desired data points
+     * @param algorithm the algorithm to use. Valid values are
+     *                  {@code "GEOMETRIC"}, {@code "LINEAR"}, and
+     *                  {@code "LESSTHANEQUAL"}. Any other value is treated as
+     *                  {@code "GEOMETRIC"}.
+     * @param assumedIntervalMillis FULL resolution is tricky because we don't
+     *                              know the period of check in question.
+     *                              Assume the minimum period and go from there.
```

Comment:
@usnavi I copied that from `granularityFromPointsGeometric`, on line 299. I don't know what the original author meant. I'm still not sure what that parameter is for.